usage/index.rst

```diff
@@ -38,6 +38,47 @@ Most of the computational power of the system is accessed through the
 batch scheduler, and so demanding applications should be submitted to it
 (see "Running Jobs").
 
+
+.. _usage-mfa:
+
+Multi Factor Authentication
+---------------------------
+
+.. note::
+
+   We are gradually rolling out MFA, so this may not yet apply to your account.
+
+When connecting to the login nodes, the system has to validate your
+identity. In order to help keep accounts secure, Bede employs a
+technique called Multi Factor Authentication, where something you
+know (your password), is combined with something you have (an app on
+a mobile device, such as your phone).
+
+When you first login to Bede, or after a password reset, you will be prompted
+to:
+
+1. Change your password
+2. Import an MFA token into an app on your mobile device
+
+Instead of `Password`, to login after this you will be prompted for:
+
+- `First Factor` - this is your password
+- `Second Factor` - this is a one-time password code provided by the token in your app
+
+Mobile apps that can be used to import an MFA token include Microsoft Authenticator,
+Google Authenticator, and others. When the token is generated on Bede, it will
```

Comment:
It might be worth adding the required protocol for "others" to make it more clear. I assume it's TOTP, but potentially HOTP based on Google Auth support.

```suggestion
Google Authenticator, and other TOTP authentication clients. When the token is generated on Bede, it will
```

If it is TOTP the relevenat RFC could be mentioned too, but TOTP will be enough info (if this is needed at all).